DigitalLearningSolutions.Data/Services/NotificationPreferenceService.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace DigitalLearningSolutions.Data.Services
+{

```

Comment:
Could we call this "NotificationPreferenceDataService"? I think we should try to go towards a pattern where all repository equivalent services are called "data service" to distinguish between the two. I'm in the process of thinking how to best separate the services and data services/repositories.